debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OpenLogReplicatorIT.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.oracle;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.sql.SQLException;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.debezium.config.Configuration;
+import io.debezium.connector.oracle.util.TestHelper;
+import io.debezium.data.VerifyRecord;
+import io.debezium.embedded.AbstractConnectorTest;
+import io.debezium.util.Testing;
+
+/**
+ * @author Chris Cranford
+ */
+public class OpenLogReplicatorIT extends AbstractConnectorTest {
+
+    private OracleConnection connection;
+
+    @Before
+    public void before() throws SQLException {
+        connection = TestHelper.testConnection();
+
+        setConsumeTimeout(TestHelper.defaultMessageConsumerPollTimeout(), TimeUnit.SECONDS);
+        initializeConnectorTestFramework();
+        Testing.Files.delete(TestHelper.SCHEMA_HISTORY_PATH);
+    }
+
+    @After
+    public void after() throws SQLException {
+        stopConnector();
+        if (connection != null && connection.isConnected()) {
+            connection.close();
+        }
+    }
+
+    @Test
+    public void shouldStreamChanges() throws Exception {
```

Comment:
Is this running for all adapters?

Reply:
Ah, this should not have been added, as it was an early basic test for OLR.  Removed.